package-lock.json

```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "@linkurious/schematics",
-      "version": "1.0.2",
+      "version": "1.0.3",
```

Comment:
why this change only in package-lock?

Reply:
it was already updated in package.json , version file ....
it was updated automatically in package-lock by running npm i